ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml

```diff
@@ -1042,7 +1042,7 @@ tests:
       BASE_DOMAIN: origin-ci-int-aws.dev.rhcloud.com
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-shared-vpc-phz
-- as: e2e-agent-compact-ipv4-conformance
+- as: e2e-agent-compact-fips
```

Comment:
Since the other two kind of agent jobs include the networking used (ie, `e2e-agent-ha-dualstack` and `e2e-agent-sno-ipv6`), what about using `e2e-agent-compact-ipv4-fips`?

Reply:
Modifiers to a job name only indicate where it deviates from defaults (other than CNI, which we always state).  IPv4 isn't needed, other jobs don't include it.

Reply:
https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/

> Job names should indicate to the reader the purpose of the specific job, as well as noting where the job deviates from OpenShift defaults for that release.

Reply:
Ack, just saw the now in the "Other deviations" section